llm4pddl/approaches/base_approach.py

```diff
@@ -14,6 +14,7 @@ class BaseApproach(abc.ABC):
 
     def __init__(self) -> None:
         self._rng = np.random.default_rng(FLAGS.seed)
+        self.embeddings_mapping: List[Dict] = []
```

Comment:
general suggestion: add an `Embedding` type in `structs.py` and use it in all of the type annotations